### PR TITLE
Guides for productivity tools.

### PR DESCRIPTION
**Pull request by chanduvkp**
Docs for productivity tools

Types of changes
----------------

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)
- Code cleanup / Refactoring
- Documentation

Description
-----------

Motivation and Context
----------------------

How Has This Been Tested?
-------------------------

Checklist:
----------

-  [] I have updated the documentation accordingly
-  [] I have written unit tests

Fixes request-yo-racks/docs#<issue_number>

### CODE REVIEW

File: ryr-docs/docs/guides/git-extras.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Install GIT extras
+
+1. Open Terminal
+2. Change the current working directory to your local project.
```
Comment (rgreinho) [inline ryr-docs/docs/guides/git-extras.md]: This step is not required.

File: ryr-docs/docs/guides/install-hub.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Hub installation
+
+1. Install hub
```
Comment (rgreinho) [inline ryr-docs/docs/guides/install-hub.md]: This step cannot happen first.

File: ryr-docs/docs/guides/install-liquidprompt.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Liquidprompt installation
+
+1. Install liquidprompt
```
Comment (rgreinho) [inline ryr-docs/docs/guides/install-liquidprompt.md]: This step cannot happen first.

**Comment by chanduvkp** [comment]
Changes are done as review. sent a PR

File: ryr-docs/mkdocs.yml [hunk 4]
```diff
@@ -9,6 +9,9 @@ pages:
   - Guides:
     - Setting up a local environment: 'guides/setup-full-environment.md'
     - Keeping your branch up-to-date: 'guides/update-branch.md'
+    - Git extras Installation: 'guides/git-extras.md'
+    - Hub Installation: 'guides/install-hub.md'
+    - Liquidprompt Installation: 'guides/install-liquidprompt.md'
```
Comment (rgreinho) [inline ryr-docs/mkdocs.yml]: The titles don't match the document titles. They should also be `Installing Liquidprompt`, etc.

**Pull request merged**